contracts/sfc/SFC.sol

```diff
@@ -288,6 +286,7 @@ contract SFC is Initializable, Ownable, Version {
         getRedirectionRequest[from] = to;
     }
 
+    // Finish redirection request and redirect address to another address
```

Comment:
Accept redirection proposal. Redirection must by accepted by the validator key holder.